app/src/main/java/com/limelight/AppView.java

```diff
@@ -386,6 +398,20 @@ public void run() {
             case CANCEL_ID:
                 return true;
 
+            case VIEW_DETAILS_ID:
+                AlertDialog.Builder builder = new AlertDialog.Builder(this);
+                builder.setMessage("App Id: " + app.app.getAppId())
```

Comment:
We should put this string into strings.xml for localization. Also please format the string as `"App ID: "`

Reply:
Whoops, yeah missed that one. Should something similar be done for the `toString` method of `ComputerDetails`? Or would you consider that out-of-scope for this PR?

Reply:
I'd say that's out of scope.